examples/player/simid_player.js

```diff
@@ -115,10 +115,19 @@ class SimidPlayer {
    * a session.
    */
   initializeAd() {
+
+    if (!this.isLinearAd_ && !this.isValidDimensions_(this.getCreativeDimensions_())) {
+      console.log('Dimensions bigger than player');
```

Comment:
You could consider using alert here, but i think this is a case where it might be appropriate. To the casual user they won't know why their click was a no op.

Maybe be more verbose message would be good here too. "Unable to play a non linear ad with dimensions bigger than the player. Please modify dimensions to a smaller size."

Reply:
I would just make the log more verbose, I don't think we should include alerts because another pop up seems more intrusive